pkg/network/server.go

```diff
@@ -512,10 +512,17 @@ func (s *Server) run() {
 			if s.peers[drop.peer] {
 				delete(s.peers, drop.peer)
 				s.lock.Unlock()
-				s.log.Warn("peer disconnected",
-					zap.Stringer("addr", drop.peer.RemoteAddr()),
-					zap.Error(drop.reason),
-					zap.Int("peerCount", s.PeerCount()))
+				if errors.Is(drop.reason, errInvalidInvType) {
```

Comment:
I'd like to add one more error to this list: the error when a node receives a message with unknown type. In general, it's not a warning itself, but almost all the nodes in the Neo network have "proper" implementation and this case may be alarming.

Reply:
I'll attach the exactly error once my node is sync up with testnet.

Reply:
> I'll attach the exactly error once my node is sync up with testnet.

OK, not this time, the peer emitting wrong messages is gone and I haven't saved the exact message to find it in our codebase.